ui/ui-frontend/package.json

```
@@ -3,19 +3,19 @@
   "version": "5.4.0-SNAPSHOT",
   "license": "MIT",
   "scripts": {
-    "analyze:archive-search": "ng build archive-search --stats-json --configuration production ; webpack-bundle-analyzer dist/archive-search/stats.json",
-    "analyze:collect": "ng build collect --stats-json --configuration production ; webpack-bundle-analyzer dist/collect/stats.json",
-    "analyze:demo": "ng build demo --stats-json --configuration production ; webpack-bundle-analyzer dist/demo/stats.json",
-    "analyze:design-system": "ng build design-system --stats-json --configuration production ; webpack-bundle-analyzer dist/design-system/stats.json",
-    "analyze:identity": "ng build identity --stats-json --configuration production ; webpack-bundle-analyzer dist/identity/stats.json",
-    "analyze:ingest": "ng build ingest --stats-json --configuration production ; webpack-bundle-analyzer dist/ingest/stats.json",
-    "analyze:pastis": "ng build pastis --stats-json --configuration production ; webpack-bundle-analyzer dist/pastis/stats.json",
-    "analyze:portal": "ng build portal --stats-json --configuration production ; webpack-bundle-analyzer dist/portal/stats.json",
-    "analyze:referential": "ng build referential --stats-json --configuration production ; webpack-bundle-analyzer dist/referential/stats.json",
+    "analyze:all": "npm run analyze:archive-search && npm run analyze:collect && npm run analyze:design-system && npm run analyze:identity && npm run analyze:ingest && npm run analyze:pastis && npm run analyze:portal && npm run analyze:referential",
```

On pourrait surement simplifier par :

```suggestion
    "analyze": "concurrently 'npm:analyze:*'",
```

---

Done.